### ISSUE TITLE
07 Remove Bookmark

### ISSUE THREAD

**Issue by daphnejong** [FroeMic/FullStack_Info_FinalProject#9]
As a **RegisteredUser**
I can **remove bookmarks from my reading list**
so that **I am not distracted by them**

**Comment by FroeMic** [comment]
If you ask yourself how to remove a bookmark, have a look at the following implementation:

In *routes.py*
```
@app.route('/trips/<trip_id>, methods = ['DELETE'])
@login_required
def delete_trip(trip_id):
    trip = Trip.query.get(trip)

    if trip is not None and trip.user.id == current_user.id:
         db.session.delete(trip)
         db.session.commit()

    flash('Trip was deleted!', 'info')
    return jsonify(
        redirect=True,
        redirect_url=url_for('trips')
    )
```

In *app.js*
```
$(document).ready(function(){
	setup();
});

function setup() {
    $('.delete-form').on('submit', submitDeleteRequest);
}

function submitDeleteRequest(e) {
    e.preventDefault();
    var endPoint = $(this).attr('action');

    $.ajax(endPoint, {
        type: 'DELETE',
        success: redirectOnSuccess
    });

}

function redirectOnSuccess (response) {
    if (response && response.redirect) {
        window.location = response.redirect_url;
    }
}
```

**Comment by FroeMic** [comment]
Implemented in 179fb31
